test/user.test.js

```diff
@@ -6,16 +6,55 @@ const UserModel = require('../dataLayer/model/user.js');
 
 /* eslint-disable no-undef */
 beforeAll(async () => {
-  /* eslint-enable no-undef */
   await mongoose.connect(global.__MONGO_URI__);
 });
 
-/* eslint-disable no-undef */
 afterAll(async () => {
-  /* eslint-enable no-undef */
   await mongoose.disconnect();
 });
 
+// language=GraphQL
+const query = `
+    query { 
+        users {
+            name
+            email
+        }
+    }
+  `;
+
+const rootValue = {};
+
+it('should return null users and authentication error without a token', async () => {
+  const result = await graphql(
+    graphqlSchema,
+    query,
+    rootValue,
+    global.mockedContextWithOutToken
+  );
+
+  expect(result.data.users).toBe(null);
+  expect(result.errors).toHaveLength(1);
+  expect(result.errors[0].message).toBe(
+    'You must supply a JSON Web Token for authorization!'
```

Comment:
Testing without a token and without a token is probably not what we want for each query and mutation. I'm happy to refactor that out into specific auth tests while reviewing this PR alternatively we wait until we have more queries to test.